src/components/grid-aware/Footer/Footer.js

```diff
@@ -0,0 +1,51 @@
+import PropTypes from "prop-types";
+import React from "react";
+import { InputTextPropType } from "../../inline/InputText";
+import s from "./Footer.module.css";
+import InfoBlock from "./InfoBlock";
+import {
+  Navigation,
+  FooterNavigationPropType,
+  SealsPropType,
+  ShelterTechLogoPropType,
+  SocialMediaLinksPropType,
+} from "./NavigationBlock";
+import SubscriptionBlock from "./SubscriptionBlock";
+
+const Footer = ({
+  inputText,
+  footerNavigation,
+  seals,
+  shelterTechLogo,
+  socialMediaLinks,
+  address,
+  employeeIdentificationNumber,
```

Comment:
It's actually employ<i>er</i> identification number, not employ<i>ee</i>. It identifies the employer (ShelterTech), not individuals who are employed by ShelterTech.

```suggestion
  employerIdentificationNumber,
```

Reply:
ok